gammapy/modeling/models/tests/test_cube.py

```diff
@@ -650,15 +650,37 @@ def test_sky_point_source():
 @requires_data()
 def test_fermi_isotropic():
     filename = "$GAMMAPY_DATA/fermi_3fhl/iso_P8R2_SOURCE_V6_v06.txt"
-    model = create_fermi_isotropic_diffuse_model(filename)
-    coords = {"lon": 0 * u.deg, "lat": 0 * u.deg, "energy": 50 * u.GeV}
-
-    flux = model(**coords)
-
-    assert_allclose(flux.value, 1.463e-13, rtol=1e-3)
+    vals = np.loadtxt(make_path(filename))
+    energy = u.Quantity(vals[:, 0], "MeV")
+    values = u.Quantity(vals[:, 1], "MeV-1 s-1 cm-2")
+
+    # No extrapolation
+    model = create_fermi_isotropic_diffuse_model(
+        filename=filename,
+        interp_kwargs={"extrapolate": False, "bounds_error": False},
+    )
+    coords_inner = {"lon": 0 * u.deg, "lat": 0 * u.deg, "energy": energy[7]}
+    flux = model(**coords_inner)
+    assert_allclose(flux.value, values[7].value, rtol=1e-3)
     assert flux.unit == "MeV-1 cm-2 s-1 sr-1"
     assert isinstance(model.spectral_model, CompoundSpectralModel)
 
+    # No extrapolation with bounds_error
```

Comment:
Is this test necessary?

Reply:
I want to ensure a good coverage of options because we were missing this issue previously!